### ISSUE TITLE
Footbal match reports exercise should depend on Exceptions first exercise

### ISSUE THREAD

**Issue by jozolam** [exercism/csharp#2306]
Hello, I think that the football match exercise https://github.com/exercism/csharp/blob/main/exercises/concept/football-match-reports/.docs/instructions.md should depend on exceptions first exercise because you ask for a thrown exception. I would create PR but not sure where are dependencies defined.

**Comment by ErikSchierboom** [comment]
Good catch! They're defined here: https://github.com/exercism/csharp/blob/main/config.json#L553-L557

**Comment by vaeng** [comment]
The calculator conundrum exercises depend on the switch concept, which is the next concept after switch. To resolve this, the exception task from the football match has to be removed or the concept of exceptions should be explained in the football match exercise. I think the removal might be better, as exceptions are taught later in the syllabus and they are not needed here.

**Comment by ErikSchierboom** [comment]
>  I think the removal might be better, as exceptions are taught later in the syllabus and they are not needed here.

I think we shouldn't remove it, but instead change it. The goal of that task is to "force" students to use a `default` case. So maybe we just change it to expect them to return a default string value instead.

**Comment by vaeng** [comment]
> So maybe we just change it to expect them to return a default string value instead.

Yes, I think that would make it more streamlined

**Comment by ErikSchierboom** [comment]
Yes please!
